src/themes/default/pages/Checkout.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div id='checkout'>
+    <personal-details></personal-details>
+    <shipping></shipping>
+    <payment></payment>
+    <order-review></order-review>
+    <microcart isCheckoutMode="true"></microcart>
```

Comment:
What isCheckoutMode is for? if you want conditional layout changes for Microcart component allowing to put it in checkout page it's a bad idea. 9 think we should just use different component for this.

Reply:
Ok, so i believe we should create /Checkot/Microcart which will have Microcart/Microcart among the mixins. WDYT?